release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -1125,6 +1125,84 @@ With this release, the CCO no longer checks for the nonexistent permission.
+
+* Previously, the *Type* column was not first on the condition table on the *MachineHealthCheck detail* page. With this release, the *Type* is now listed first on the condition table. (link:https://issues.redhat.com/browse/OCPBUGS-27246[*OCPBUGS-27246*])
+
+* Previously, the console plugin proxy was not copying the status code from plugin service responses. This caused all responses from the plugin service to have a `200` status, causing unexpected behavior, especially around browser caching. With this release, the console proxy logic was updated to forward the plugin service proxy response status code. Proxied plugin requests now behave as expected. (link:https://issues.redhat.com/browse/OCPBUGS-26933[*OCPBUGS-26933*])
+
+* Previously, when cloning a persistent volume claim (PVC), the modal assumes pvc.spec.resources.requests.storage value includes a unit. With this release, `pvc.spec.resources.requests.storage` includes a unit suffix and the *Clone PVC* modal works as expected. (link: https://issues.redhat.com/browse/OCPBUGS-26772[*OCPBUGS-26772*])
+
+* Previously, escaped strings were not handled properly when editing {vmw-full} connection, causing broken {vmw-full} configuration. With this release, the escape strings work as expected and the {vmw-full} configuration no longer breaks. (link:https://issues.redhat.com/browse/OCPBUGS-25942[*OCPBUGS-25942*])
+
+* Previously, when configuring a {vmw-full} connection,  the `resourcepool-path` key was not added to the {vmw-full} config map which might have caused issues connecting to {vmw-full}. With this release, there are no longer issues connecting to {vmw-full}. (link:https://issues.redhat.com/browse/OCPBUGS-25927[*OCPBUGS-25927*])
+
+* Previously, there was missing text in the *Customer feedback* modal. With this release, the link text is restored and the correct Red Hat image is displayed. (link:https://issues.redhat.com/browse/OCPBUGS-25843[*OCPBUGS-25843*])
+
+* Previously, the *Update cluster* modal would not open when clicking *Select a version* from the *Cluster Settings* page. With this release, the *Update cluster* modal shows when clicking *Select a version*. (link:https://issues.redhat.com/browse/OCPBUGS-25780[*OCPBUGS-25780*])
+
+* Previously, the filter part in the resource section of the *Search* page did not work on a mobile device. With this release, filtering now works as expected on a mobile device. (link:https://issues.redhat.com/browse/OCPBUGS-25530[*OCPBUGS-25530*])
+
+* Previously, the console Operator was using client instead of listeners for fetching a cluster resource causing the Operator to do operations on resources with old revision. With this release, the console Operator uses list to fetch data from cluster instead of clients. (link:https://issues.redhat.com/browse/OCPBUGS-25484[*OCPBUGS-25484*])
```

Comment:
```suggestion
* Previously, the console Operator was using a client instead of listeners for fetching a cluster resource. This caused the Operator to do operations on resources with an older revision. With this release, the console Operator uses list to fetch data from cluster instead of clients. (link:https://issues.redhat.com/browse/OCPBUGS-25484[*OCPBUGS-25484*])
```